pkg/util/client.go

```diff
@@ -171,3 +171,13 @@ func (c *Client) SearchTraceQL(query string) (*tempopb.SearchResponse, error) {
 
 	return m, nil
 }
+
+func (c *Client) SearchTraceQLWithRange(query string, start int64, end int64) (*tempopb.SearchResponse, error) {
+	m := &tempopb.SearchResponse{}
+	_, err := c.getFor(c.BaseURL+"/api/search?q="+url.QueryEscape(query)+"&start="+strconv.FormatInt(start, 10)+"&end="+strconv.FormatInt(end, 10), m)
```

Comment:
i think there are more correct ways to build a query string like this. likely using the url object and then asking it to encode all at once, but this is fine given what we're doing here